src/data/real/ennreal.lean

```diff
@@ -84,7 +84,9 @@ localized "notation `ℝ≥0∞` := ennreal" in ennreal
 localized "notation `∞` := (⊤ : ennreal)" in ennreal
 
 noncomputable instance : linear_ordered_add_comm_monoid ℝ≥0∞ :=
-{ .. ennreal.canonically_ordered_comm_semiring,
+{ max_def := rfl,
+  min_def := rfl,
```

Comment:
I have no idea why `rfl` works here but the `reflexivity` `auto_param` does not. Either way, I'm inclined not to care. I can leave a TODO comment if someone else cares.

Reply:
These lines create a new diamond: now `max` on `ennreal` is not defeq `sup`, so we'll have two `sup`s. Please derive `linear_order` and use it here. Or just use `.. (infer_instance : linear_order ennreal)`.

Reply:
Oops, I must not have been paying attention. `derive` is the right choice here.